### DISCUSSION TITLE
[Feature] Sorting by “least recent photo” in Album view

### DISCUSSION THREAD

**Discussion by hachre** [immich-app/immich#3962] [Feature Request]
### The feature

Hi,

I saw that the entire sorting feature is relatively new and recently got introduced here:
https://github.com/immich-app/immich/pull/2861

I would love to see one more option next to “Most recent photo” which should be “Least recent photo”. I often like to look at photos within an Album in their chronological order.

Thank you

### Platform

- [ ] Server
- [x] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1689
